### PR TITLE
FinestraMiner changed specs

### PR DESCRIPTION
**Pull request by IrfanCALISKAN**

### CODE REVIEW

**Comment by cokes518** [comment]
@tyler-whitman @matthewcarlreetz can you review?

File: src/makers/finestra/finestra.svg [hunk 1]
```diff
@@ -1,40 +1 @@
...
-		"/>
-	<path class="st2" d="M52.7,38.9c0,0.6-0.2,1.2-0.7,1.6s-1,0.7-1.6,0.7c-0.6,0-1.2-0.2-1.6-0.7c-0.5-0.5-0.7-1-0.7-1.6
-		c0-0.6,0.2-1.2,0.7-1.6c0.5-0.5,1-0.7,1.6-0.7c0.5,0,0.9,0.1,1.3,0.4v-0.4h1V38.9z M51.3,39.8c0.3-0.3,0.4-0.6,0.4-0.9
-		c0-0.4-0.1-0.7-0.4-0.9s-0.6-0.4-0.9-0.4c-0.4,0-0.7,0.1-0.9,0.4S49,38.5,49,38.9c0,0.4,0.1,0.7,0.4,0.9s0.6,0.4,0.9,0.4
-		C50.7,40.2,51,40.1,51.3,39.8z"/>
-</g>
-<rect x="3" y="2.7" class="st3" width="69.2" height="70.4"/>
-</svg>
+<svg id="katman_1" data-name="katman 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 801.49 807.63"><defs><style>.cls-1{fill:none;stroke:#000;stroke-miterlimit:10;stroke-width:18.98px;}</style></defs><rect class="cls-1" x="9.49" y="9.49" width="782.51" height="788.65" rx="204.49"/><path d="M301.08,293.35H503.21a0,0,0,0,1,0,0v18A36.18,36.18,0,0,1,467,347.53H301.08a0,0,0,0,1,0,0V293.35A0,0,0,0,1,301.08,293.35Z"/><path d="M301.08,374H427.74a0,0,0,0,1,0,0v18a36.18,36.18,0,0,1-36.18,36.18H301.08a0,0,0,0,1,0,0V374A0,0,0,0,1,301.08,374Z"/><path d="M301.08,454.74h50.68a0,0,0,0,1,0,0v20.34a33.84,33.84,0,0,1-33.84,33.84H301.08a0,0,0,0,1,0,0V454.74A0,0,0,0,1,301.08,454.74Z"/></svg>
```
Comment (matthewcarlreetz) [inline src/makers/finestra/finestra.svg:1]: Your path needs to have a fill color like this
```
<svg id="katman_1" data-name="katman 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 801.49 807.63">
  <defs>
    <style>.cls-1{fill:none;stroke:#000;stroke-miterlimit:10;stroke-width:18.98px;}</style>
  </defs>
  <rect class="cls-1" x="9.49" y="9.49" width="782.51" height="788.65" rx="204.49" />
  <path fill="#000" d="M301.08,293.35H503.21a0,0,0,0,1,0,0v18A36.18,36.18,0,0,1,467,347.53H301.08a0,0,0,0,1,0,0V293.35A0,0,0,0,1,301.08,293.35Z" />
  <path fill="#000" d="M301.08,374H427.74a0,0,0,0,1,0,0v18a36.18,36.18,0,0,1-36.18,36.18H301.08a0,0,0,0,1,0,0V374A0,0,0,0,1,301.08,374Z" />
  <path fill="#000" d="M301.08,454.74h50.68a0,0,0,0,1,0,0v20.34a33.84,33.84,0,0,1-33.84,33.84H301.08a0,0,0,0,1,0,0V454.74A0,0,0,0,1,301.08,454.74Z" />
</svg>
```

**Review by matthewcarlreetz — CHANGES_REQUESTED** [review]
The svg needs a little fix, otherwise looks good.

File: src/makers/finestra/finestra.svg (hunk 1 above)
Comment (IrfanCALISKAN) [reply src/makers/finestra/finestra.svg]: Thx Matthew, I added svg path fill color in the new commit.

**Review by matthewcarlreetz — APPROVED** [review]

**Pull request merged**